configs/components/rubygem-net-netconf.rb

```diff
@@ -1,17 +1,30 @@
 component "rubygem-net-netconf" do |pkg, settings, platform|
   pkg.version "0.4.3"
-  pkg.md5sum "fa173b0965766a427d8692f6b31c85a4"
-  pkg.url "http://buildsources.delivery.puppetlabs.net/net-netconf-#{pkg.get_version}.gem"
+  if platform.is_huaweios?
+    pkg.url "http://buildsources.delivery.puppetlabs.net/net-netconf-#{pkg.get_version}.tar.gz"
```

Comment:
This is the same situation as nokogiri - it's an archive from a native install. I had to do this because if you try to install this gem on an x86 server, the install process notes that no x86 extensions exist for nokogiri, and it blows up trying to build nokogiri again.